manifests/plugin/remote_execution/ssh/params.pp

```diff
@@ -0,0 +1,10 @@
+# Remote Execution SSH default parameters
+class foreman_proxy::plugin::remote_execution::ssh::params {
+  $enabled           = true
+  $listen_on         = 'https'
+  $generate_keys     = true
+  $ssh_identity_dir  = "${::foreman_proxy::dir}/.ssh"
```

Comment:
I see a few options:

* ```include foreman_proxy``` and use ```$::foreman_proxy::dir```
* ```include foreman_proxy::params``` and use ```$::foreman_proxy::params::dir```
* ```include foreman_proxy::params``` and use ```pick($::foreman_proxy::dir, $::foreman_proxy::params::dir)```
* Use undef and do the pick in ssh.pp

I'm leaning to the third option.